### PR TITLE
Week five sorting bug fix

### PR DESCRIPTION
**Pull request by EmmaWenzel**

### CODE REVIEW

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java [hunk 1]
```diff
@@ -419,6 +419,36 @@ public void optionalAttendeeAvailable() {
...
+        new MeetingRequest(Arrays.asList(PERSON_A, PERSON_B, PERSON_C), DURATION_30_MINUTES);
+ 
+    Collection<TimeRange> actual = query.query(events, request);
+    Collection<TimeRange> expected =
+        Arrays.asList(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, TIME_0800AM, false),
+            TimeRange.fromStartEnd(TIME_0930AM, TimeRange.END_OF_DAY, true));
+ 
+    Assert.assertEquals(expected, actual);
+  }
+
```
Comment (ofk-goog) [inline walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:451]: We need more test cases to uncover the bug. For starters, try this test case (given to the test in order from top to bottom)

```
     |---A--------------|
                                                  |----B---|
             |-----C---|
                                             |---------D--------|
```
where time is left-to-right.

Comment (EmmaWenzel) [reply walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:451]: There's already a test that looks like this:
// Events  :        |----A----|
//                         |--B--|
// Day       : |---------------------|
// Options : |--1--|          |--2--|

I think that would have similar results to the A/C test, but if not let me know and I'll throw a test like that in (or just revise that test to have the times in a different order?)

I'll add a test for the B/D scenario though

Comment (EmmaWenzel) [reply walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:451]: (that comment didn't quite transfer the B was supposed to be nested within a)
But I just realized you wanted 4 different events in the same test case, given in the order A, B, C , D where C is nested in A and B is nested in D, right?

Comment (ofk-goog) [reply walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:451]: Right.

I think another test that might fail right now is simply: 2 events, in reverse chronological order, with enough space in between for the to-be-scheduled event. Unless I'm misunderstanding your code.

Comment (EmmaWenzel) [reply walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:452]: Okay so the test I wrote did pass. Let me know if that wasn't what you meant though

Comment (ofk-goog) [reply walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:452]: Yeah it turns out I was wrong this whole time. Your code works great. Sorry about that.

**Pull request merged**